zyte_api/aio/client.py

```diff
@@ -18,9 +18,8 @@
 from ..stats import AggStats, ResponseStats
 from ..utils import user_agent
 
-AIO_API_TIMEOUT = aiohttp.ClientTimeout(total=API_TIMEOUT + 60,
-                                        sock_read=API_TIMEOUT + 30,
-                                        sock_connect=10)
+
+AIO_API_TIMEOUT = aiohttp.ClientTimeout(total=API_TIMEOUT + 120)
```

Comment:
Removing `sock_read` and `sock_connect`. A single total timeout seems a good idea here. If the client requires more interactive timeouts here, he can change them. 

Increasing the timeout a bit more. Default aiohttp is 5 minutes. Here we are setting it to 3 minutes.

Reply:
@ivanprado after re-checking the code, it would seem that `API_TIMEOUT` is only being used in this module and is currently being set to a value of **60** in `constants.py`.

It might be better if we increase the said timeout directly instead of adding `+ 120` here. Moreover, all future timeout constants might be better if they're defined in `constants.py` directly and are simply imported.

Reply:
The idea of API_TIMEOUT constant is that it is the same as the timeout we have on a server side. On a client side we add a bit more, to account for various scenarios, and we're explicit in how much we're increasing it.

Reply:
I.e. the API should return an error response after API_TIMEOUT time is spent processing the request, and the extra 60s here is to account for various things which may happen; I think it was quite conservative.